quickstarts/istio/config.yml

```diff
@@ -0,0 +1,56 @@
+# Name of the quickstart (required)
+name: istio-quickstart
+
+# Displayed in the UI (required)
+title: Istio Service Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  This dashboard will give you insight into services and applications running
+  in Kubernetes clusters with an Istio Service Mesh enabled. Includes client/server focused service metrics,
+  and an additional page for displaying Ingress Gateway metrics.
+
+  The only requirement for installing this along with the New Relic Kubernetes agent is the
+  Prometheus OpenMetrics Integration. [POMI](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic)
+
+  [Istio Standard Metrics](https://istio.io/latest/docs/reference/config/metrics/)
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A dashboard view of services running in your Kubernetes cluster using an Istio Service Mesh.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
```

Comment:
Is this officially supported by New Relic? Does it match our level criteria for support?

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#support-levels

Reply:
Right now New Relic is the only one supporting this dashboard.